run.c

```diff
@@ -194,7 +194,10 @@ void softmax(float* x, int size) {
 void matmul(float* xout, float* x, float* w, int n, int d) {
     // W (d,n) @ x (n,) -> xout (d,)
     // by far the most amount of time is spent inside this little function
+    #pragma omp target data map(tofrom: w[0:d*n],x[0:n], xout[0:d])
+    {
     int i;
+    #pragma omp target device (0)
```

Comment:
cannot these 2 pragmas be joined into 1?
the `int i` can be removed, and use `for (int i = 0; i < d; i++) {`
also removing the `private(i)`